bedrock/mozorg/templates/mozorg/contribute/tasks/taskview-base.html

```diff
@@ -0,0 +1,49 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/base-resp.html" %}
+
+{% block body_class %}taskview{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'contribute-taskview' %}
+{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+{% block site_header_logo %}{% endblock %}
+
+{% block alt_header %}
+  <h1>{{ _('Mozilla Participation Tasks') }}</h1>
+  <a href="{% block more_tasks %}{{ url('mozorg.contribute.signup') }}{% endblock %}" class="back">{{ _('I want to pick a different task.' )}}</a>
```

Comment:
Should this `more_tasks` URL have `?v=1` appended? Currently on all v1 tasks, clicking "I want to pick a different task" takes you to the current/existing signup page.